### DISCUSSION TITLE
CallBack function on modal close for Android

### DISCUSSION THREAD

**Discussion by omarnaol** [azeezat/react-native-select#71] [Q&A] [answered]
Hi!

Thanks for the package, really useful so far!

I need to detect when the modal is being closed and pass a callback function. I found this option (onDismiss), but it works just with IOS. How could I manage it with Android?

        modalControls={{
          modalProps: {
            onDismiss: () => console.log("ondismiss call")

          },
        }}
        
Thanks in advance!!!

**Comment by azeezat** [comment]
@omarnaol I'm glad you have found this library helpful. The onDrismiss prop on the modal works only with iOS according to the [react native docs](https://reactnative.dev/docs/modal#ondismiss-ios). We would look into having our modal support it on Android in the next release. Thanks again for reporting this issue.

**Reply by azeezat** [reply, marked as answer]
@omarnaol A new [release 1.3.7](https://www.npmjs.com/package/react-native-input-select) has just been published. You don't need to update your code, just test that the functionality works on your android device. Please let me know if this helps.

**Reply by omarnaol** [reply]
Hi azeezat

It seems to work perfectly, thank you very much, I appreciate it!!!!!!

**Reply by azeezat** [reply]
Happy to help
